tensorboard/plugins/graph/tf_graph_loader/tf-graph-dashboard-loader.html

```diff
@@ -28,6 +28,6 @@
 `selection` from the tf-graph-controls as an input as opposed to URL path of an
 data endpoint.
 -->
-<dom-module id="tf-graph-dashboard-loader"> </dom-module>
+<dom-module id="tf-graph-dashboard-loader"><template></template></dom-module>
```

Comment:
Can we add a comment about why we need the empty template?  Like  `<!-- strictTemplatePolicy requires a template (even a null one) -->`